ports/esp32/boards/GENERIC_C3/sdkconfig.c3usb

```diff
@@ -1,4 +1,3 @@
-CONFIG_ESP32C3_REV_MIN_3=y
```

Comment:
why is this line removed?

Reply:
The simple answer is because it's the default anyway (without a bit of stuffing around you cannot compile firmware for revisions before 3).

I tried to piece the history together but I think what happened was that the engineering samples (rev 1 & 2) didn't support the USB peripheral, but our original GENERIC_C3 board definition was added then, and then later the GENERIC_C3_USB board was added to support the production boards.

Then later (IDF 5.0 I think, but haven't confirmed) the IDF made revision 3 the default minimum. i.e. neither GENERIC_C3 and GENERIC_C3_USB builds work by default any more on engineering samples. (I know this because I _only_ have engineering sample c3 boards).

So the longer answer is that I think we should actually switch this around and make usb the default (or possibly just not have a variant at all).

Reply:
Hmm, OK.  So maybe we should follow IDF 5 and have the default GENERIC_C3 variant be minimum rev 3 and enable USB.  Then we could have a variant which runs on < rev 3 without USB.  Is that possible/easy?  If not easy then we can just have no variants for GENERIC_C3 and enable USB by default (until someone asks for < rev 3 support).

Said differently: if we only support rev 3, then we should enable USB by default.

Reply:
Yes, I agree.

Will update later tonight.

Reply:
Done.